src/meta/src/hummock/manager/mod.rs

```diff
@@ -1063,6 +1087,85 @@ where
         Ok(())
     }
 
+    /// We don't commit an epoch without checkpoint. We will only update the `max_current_epoch`.
+    #[named]
+    pub async fn update_current_epoch(&self, max_current_epoch: HummockEpoch) -> Result<()> {
```

Comment:
The logic in this function shares a lot of similarities with `commit_epoch`. Can we extract the common logic (i.e. update version) into a function and let `update_current_epoch` and `commit_epoch` use it to avoid duplicate codes?